### ISSUE TITLE
Ad skipping not working since the lastest chrome update.

### ISSUE THREAD

**Issue by Starcraft426** [proffapt/own-youtube#44]
**Describe the bug**
The function which is supposed to skip the videos by clicking on the button seems to not work anymore. In fact, since the last update of chome, it as changed his look and youtube's one too. This has probably modified the classes or id of the button, making the function unable to work properly. 
```javascript
Array.from(document.querySelectorAll('.ytp-ad-overlay-close-button'))?.forEach(e => e?.click());
``` 
could probably be replaced with
```javascript
Array.from(document.querySelectorAll('.ytp-ad-skip-button-modern'))?.forEach(e => e?.click());
```
knowing that 'ytp-ad-skip-button-modern' is the name of the button when selected on the devtool.

**Comment by proffapt** [comment]
Did you try changing the name and test it?

**Comment by Starcraft426** [comment]
Seems to be working for me

**Comment by Starcraft426** [comment]
Just noticed that it seems like the ads are not playing anymore with this little fix. Normally, ads play for like 5 seconds before being skipped but the way the button is put in the page, probably even before the ad, it is pressed instantly and the ad don't play. What a great update

**Comment by proffapt** [comment]
would appreciate you to create a pr

**Comment by proffapt** [comment]
resolved in #45
